modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenConstants.java

```diff
@@ -82,4 +82,9 @@
 
     public static enum MODEL_PROPERTY_NAMING_TYPE {camelCase, PascalCase, snake_case, original}
 
+    public static final String MODEL_NAME_PREFIX = "modelNamePrefix";
+    public static final String MODEL_NAME_PREFIX_DESC = "Prefix that will be prepended to all model names. Default is the empty string.";
```

Comment:
That's true for `CliOption`, but I don't think it would work the same for maven/airline cli options, e.g.:

```
 @Option(name = {"--model-name-suffix"}, 
         title = "model name suffix", 
         description = CodegenConstants.MODEL_NAME_SUFFIX_DESC)
```

Airline, for example, only has `allowedValues`.